config/database2.yml

```diff
@@ -0,0 +1,17 @@
+development:
```

Comment:
We should remove this file, look that you have two database.yml and only the first one is the correct.
There you have the right username and pasword